docs/source/user_guide/computed_properties.rst

```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
 
 .. _mobility:
 
-.. Charge Carrier Mobility
-.. -----------------------
+Charge Carrier Mobility
+-----------------------
+
+Charge carrier mobility is a measure of the speed at which charge carriers, such as electrons or holes,
+can move through a material when an electric field is applied. It reflects the material's ability to conduct charge and
+is influenced by various intrinsic and extrinsic factors.
+
+
```

Comment:
Should we stick to the pattern of the rest of the properties and do 

Property description
^^^^^^^^^^^^^^^^^^^^^
[everything up to Output files] 

Output files
^^^^^^^^^^^^^^^

I think the levels of headers are === --- ^^^ ~~~~

Reply:
fixed